framework/app.mk

```diff
@@ -108,7 +108,7 @@ endif
 
 # header files
 include_dirs	:= $(shell find $(depend_dirs) -type d | grep -v "\.svn")
-app_INCLUDE     := $(foreach i, $(include_dirs), -I$(i)) $(ADDITIONAL_INCLUDES)
+include_files	:= $(shell find $(depend_dirs) -name "*.[hf]" | grep -v "\.svn")
```

Comment:
We do not need the SVN part I think - it is probably copy'n'paste of the above line?